setup.py

```diff
@@ -77,8 +77,10 @@ def run(self, *arg, **kw):
     'geospatial': ['geojson'],
     'thumbnails': ['Pillow', 'pydicom', 'numpy'],
     'worker': ['celery'],
-    'plugins': ['celery', 'geojson', 'Pillow', 'pydicom', 'numpy']
+    'oauth': ['pyjwt', 'cryptography']
 }
+all_extra_reqs = [y for x in extras_reqs.values() for y in x]
```

Comment:
I find this construct difficult to read, would you mind changing to:

    all_extra_reqs = itertools.chain.from_iterable(extras_reqs.values())